style.css

```diff
@@ -399,31 +406,166 @@ form > * {
   color: #ebebff;
 }
 
-form > input:focus {
-  border: 2px solid #9490f5;
+.desk {
+  display: none;
+}
+
+.elementRight-1 {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .toolbar {
+    max-height: 72px;
+    max-width: 100%;
+    justify-content: space-evenly;
+  }
+
+  .logo {
+    line-height: 20px;
+  }
+
+  .nav > .button-nav {
+    display: none;
+  }
+
+  nav > ul {
+    display: flex;
+    gap: 8px;
+  }
+
+  nav > ul > .nav-link1 {
+    padding-bottom: 8px;
+  }
+
```

Comment:
- [ ] Your navigation links look great 👏. In addition to what you have done so far, I would suggest you remove the underlines on your links. Look at the comparison below 👇 
**Your work**
![image](https://user-images.githubusercontent.com/90222110/177837705-aacc7d76-b224-4f88-bfd1-7c1c02629adf.png)
**Figma design**
![image](https://user-images.githubusercontent.com/90222110/177837855-e448a8b0-7d6b-44da-a334-06422f10c68e.png)

💡  **HINT**: You may consider adding `text-decoration:none` to the anchor tags `<a>` in your nav list items

- [ ] It would be much better if you add more spacing between your nav links. Like increasing the gap to around 30px because as you can see on the Figma design there is more white-spaces between them.😃 👍